tests/model_serving/model_server/runtime_configuration/conftest.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Generator
+
+import pytest
+from _pytest.fixtures import FixtureRequest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.namespace import Namespace
+from ocp_resources.secret import Secret
+from ocp_resources.serving_runtime import ServingRuntime
+
+from utilities.inference_utils import create_isvc
+
+
+@pytest.fixture(scope="class")
+def s3_models_second_inference_service(
```

Comment:
Why is it called "second" inference service? Where is the first and what is the difference? 
Can't you parametrize a single fixture to create multiple ISVCs?

Reply:
first one is using a generic isvc fixture; can't call same fixture twice with different params in the same test